gpustack/assets/model-catalog.yaml

```diff
@@ -722,10 +722,38 @@ model_sets:
   specs:
     - mode: standard
       quantization: "BF16"
+      gpu_filters:
+        vendor: nvidia
       source: huggingface
       huggingface_repo_id: black-forest-labs/FLUX.1-dev
       backend: SGLang
-      backend_version: 0.5.5
+      backend_version: 0.5.6.post2
+      env:
+        GPUSTACK_MODEL_VRAM_CLAIM: "37580963840" # 35 GiB, observed empirically
+- name: FLUX.2 Dev
+  description: FLUX.2 [dev] is a 32 billion parameter rectified flow transformer capable of generating, editing and combining images based on text instructions.
+  home: https://blackforestlabs.ai
+  icon: /static/catalog_icons/blackforestlabs.png
+  size: 32
+  categories:
+    - image
+  licenses:
+    - flux-dev-non-commercial-license
+  release_date: "2025-11-25"
+  deployment_notes: "Due to the large model size, enable_hf_xet/--enable-hf-xet must be enabled on the worker to deploy this model."
```

Comment:
Currently, hf-xet is not enabled by default. How about we skip this model for now and add until it can work out of the box.